roles/frr/templates/frr_loadbalancer.conf.j2

```diff
@@ -36,27 +36,63 @@ router bgp {{ frr_local_as }}
 exit
 !
 ip prefix-list internal seq 5 permit {{ frr_internal_v4 }} ge 32 le 32
+{% for item in frr_internalbackup_v4 %}
+ip prefix-list internalbackup seq {{ item.seq }} permit {{ item.cidr }} ge 32 le 32
+{% endfor %}
 ip prefix-list external seq 5 permit {{ frr_external_v4 }} ge 32 le 32
+{% for item in frr_externalbackup_v4 %}
```

Comment:
frr_externalbackup_v4 has to be added to defaults.yml

Reply:
dito